src/CryptoAxus.Domain/Collections/AllEventDocument .cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace CryptoAxus.Domain.Collections;
+
+[BsonCollection("AllEvent")]
+public class AllEventDocument: BaseDocument
+{
+    [BsonElement(elementName: "nftId", Order = 2)]
+    public string NftId { get; set; }
```

Comment:
Check with the frontend team which fields are nullable and place the relevant check on it.

Reply:
Resolved